core/whistle_apps-1.0.0/src/whapps_util.erl

```diff
@@ -284,6 +285,24 @@ is_account_db(Db) -> couch_util:db_classification(Db) =:= 'account'.
 
 %%--------------------------------------------------------------------
 %% @public
+%% @doc Ids are one->one with accounts.
+%% @end
+%%--------------------------------------------------------------------
+-spec get_account_by_id(ne_binary()) ->
+                                  {'ok', ne_binary()} |
+                                  {'error', 'not_found'}.
+get_account_by_id(Id) ->
+    case couch_mgr:open_doc(?WH_ACCOUNTS_DB, Id) of
```

Comment:
Good question. It appears that there are dependents of the existing 2 functions all over the place. No particular reason why open_cache_doc wasn't used.
However, I looked at the reason this PR was included was to actually help #1250. And as it turns out I should just be using wh_util:format_account_id to produce the account db binary string. I'm going to close this one